reports/kriti21.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: report
+report_year: '2018'
+permalink: /GSoC/2018/StatusReport/KritiRohilla
+student: Kriti Rohilla
+organisation: coala
+organisation_link : https://coala.io
+project: Git Commit Content Inspection
+project_link: https://summerofcode.withgoogle.com/projects/#5636421247827968
+tarball: https://raw.githubusercontent.com/kriti21/GSoC-2018-Report/master/ProjectPatches.tar
+mentors: >
+ [Saurav Singh](https://github.com/damngamerz)
+ [Rafid Aslam](https://github.com/refeed)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/9
+ - Phase 1 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/36
+ - Phase 2 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/41
+ - Phase 3 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/42
+bio: >
+ I am in my pre-final year at University Institute of Engineering and Technology,
+ Panjab University. As part of my GSoC 2018 project, I worked on vcs related bears
```

Comment:
Capitalize `VCS` as it's an abbreviation

Reply:
And fix indentation (c.f. https://github.com/coala/projects/issues/615)

and add OpenHub social

and start activities at 1 , not 0